perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/SchedulingPool.java

```diff
@@ -38,10 +38,15 @@
 	int addToPool(Task task, DispatcherQueue dispatcherQueue)
 			throws InternalErrorException;
 
+	void addTaskSchedule(Task task, int delayCount);
```

Comment:
ack, added the javadoc, and fixed missing `addTaskSchedule()` in `reloadTasks()`.
In a nutshell `addToPool` adds the task to `SchedulingPool`, but not its queue with Tasks that are waiting to be sent to Engine, this is achieved separately by `addTaskSchedule()`.